benchmarks/benchmark_rmi_alt.cc

```diff
@@ -3,53 +3,81 @@
-#include "competitors/analysis-rmi/include/rmi/rmi.hpp"
-#include "competitors/rmi_cpp.h"
+#include "competitors/analysis-rmi/include/rmi/rmi_robust.hpp"
+#include "competitors/rmi_cpp_robust.h"
 
 template <template <typename> typename Searcher>
 void benchmark_32_rmi_cpp(sosd::Benchmark<uint32_t, Searcher>& benchmark,
                           bool pareto) {
-  benchmark.template Run<RMICpp<uint32_t, rmi::LinearSpline, rmi::LinearRegression,
-                   rmi::RmiLAbs, 128, 1>>();
+  benchmark.template Run<
+      RMICppRobust<uint32_t, rmi::LinearRegression, rmi::LinearRegression,
+                   rmi::RmiLAbsRobust, 128, 1>>();
   if (pareto) {
-    benchmark.template Run<RMICpp<uint32_t, rmi::LinearSpline, rmi::LinearRegression,
-                     rmi::RmiLAbs, 256, 2>>();
-    benchmark.template Run<RMICpp<uint32_t, rmi::LinearSpline, rmi::LinearRegression,
-                     rmi::RmiLAbs, 512, 3>>();
-    benchmark.template Run<RMICpp<uint32_t, rmi::LinearSpline, rmi::LinearRegression,
-                     rmi::RmiLAbs, 1024, 4>>();
-    benchmark.template Run<RMICpp<uint32_t, rmi::LinearSpline, rmi::LinearRegression,
-                     rmi::RmiLAbs, 2048, 5>>();
-    benchmark.template Run<RMICpp<uint32_t, rmi::LinearSpline, rmi::LinearRegression,
-                     rmi::RmiLAbs, 4096, 6>>();
-    benchmark.template Run<RMICpp<uint32_t, rmi::LinearSpline, rmi::LinearRegression,
-                     rmi::RmiLAbs, 8192, 7>>();
-    benchmark.template Run<RMICpp<uint32_t, rmi::LinearSpline, rmi::LinearRegression,
-                     rmi::RmiLAbs, 16384, 8>>();
+    benchmark.template Run<
+        RMICppRobust<uint32_t, rmi::LinearRegression, rmi::LinearRegression,
+                     rmi::RmiLAbsRobust, 512, 2>>();
+    benchmark.template Run<
+        RMICppRobust<uint32_t, rmi::LinearRegression, rmi::LinearRegression,
+                     rmi::RmiLAbsRobust, 2048, 3>>();
+    benchmark.template Run<
+        RMICppRobust<uint32_t, rmi::LinearRegression, rmi::LinearRegression,
+                     rmi::RmiLAbsRobust, 8192, 4>>();
+    benchmark.template Run<
+        RMICppRobust<uint32_t, rmi::LinearRegression, rmi::LinearRegression,
+                     rmi::RmiLAbsRobust, 32768, 5>>();
+    benchmark.template Run<
+        RMICppRobust<uint32_t, rmi::LinearRegression, rmi::LinearRegression,
+                     rmi::RmiLAbsRobust, 131072, 6>>();
+    benchmark.template Run<
+        RMICppRobust<uint32_t, rmi::LinearRegression, rmi::LinearRegression,
+                     rmi::RmiLAbsRobust, 524288, 7>>();
+    benchmark.template Run<
+        RMICppRobust<uint32_t, rmi::LinearRegression, rmi::LinearRegression,
+                     rmi::RmiLAbsRobust, 2097152, 8>>();
+    benchmark.template Run<
+        RMICppRobust<uint32_t, rmi::LinearRegression, rmi::LinearRegression,
+                     rmi::RmiLAbsRobust, 8388608, 9>>();
+    benchmark.template Run<
```

Comment:
Why did we change the model parameters?

Reply:
The old parameters were still 4 orders of magnitude smaller than the largest reference RMIs, which seemed suboptimal for comparison